static/app/views/settings/project/projectOwnership/ruleBuilder.spec.jsx

```diff
@@ -113,36 +108,32 @@ describe('RuleBuilder', function () {
     );
 
     // Open the menu so we can do some assertions.
-    openMenu(wrapper, {name: 'owners', control: true});
-    await tick();
-    wrapper.update();
+    const ownerInput = screen.getByRole('textbox', {name: 'Rule owner'});
+    selectEvent.openMenu(ownerInput);
 
-    // Should have all 4 users/teams listed
-    expect(wrapper.find('IdBadge')).toHaveLength(4);
+    await waitForElementToBeRemoved(() => screen.queryByText('Loading...'));
```

Comment:
jfyk.. sometimes I have issues with the ellipsis in my tests and check only for the loading. example: 

`screen.queryByText(/loading/)`

Reply:
👍 yeah sometimes we use …